### PR TITLE
Update diabetes-2-diagnosis.html

### PR DESCRIPTION
**Pull request by hvonhaaren**
Condensed info and fixed links

### CODE REVIEW

File: app/views/diabetes-content/diabetes-2-diagnosis.html [hunk 1]
```diff
@@ -42,21 +42,19 @@
 Then make another [GP appointment](https://nhsalpha.herokuapp.com/book-an-appointment/start) and take your list of questions with you.
 
 ###Dealing with confusion and shock after the diagnosis
-There’s also a lot of [information on diabetes](/XXX) available where you can read up on the condition.
+There’s also a lot of [information on diabetes](https://nhsalpha.herokuapp.com/diabetes-content/information-and-support-type-2-diabetes) available on:
```
Comment (Demotive) [inline app/views/diabetes-content/diabetes-2-diagnosis.html]: "available through" ?

Comment (hvonhaaren) [inline app/views/diabetes-content/diabetes-2-diagnosis.html]: Thought more as in 'info on support groups' etc on that page we link to

Comment (Demotive) [inline app/views/diabetes-content/diabetes-2-diagnosis.html]: Ah okay I see... Maybe "including" then?

Or "...a lot of information on diabetes available including:"

Sorry, doesn't seem to read right at the moment?

File: app/views/diabetes-content/diabetes-2-diagnosis.html [hunk 2]
```diff
@@ -42,21 +42,19 @@
...
 
-Some hospitals offer [diabetes courses](LINK TO INFO PAGE) where you can learn more about the condition.
-
-You can also talk to other people who have diabetes. You can do this through [online forums](/XXX) or in [support groups](https://www.diabetes.org.uk/How_we_help/Local_support_groups/).
-
-Hearing how other people manage their diabetes and what changes they had to make to their lifestyle could help you organise how you live with diabetes.
+- support groups
+- hospital courses about living with diabetes
+- online forums and chat rooms where peole with diabetes discuss how they manage their condition
```
Comment (Demotive) [inline app/views/diabetes-content/diabetes-2-diagnosis.html]: people

Comment (Demotive) [inline app/views/diabetes-content/diabetes-2-diagnosis.html]: typo - "people" not "peole"

**Pull request merged**